docs/enroute/Melbourne Centre/PIY.md

```diff
@@ -60,4 +50,22 @@ Hyden (HYD) is responsible for assigning and issueing arrival clearance to aircr
 ### Jarrah
 Jarrah (JAR) is responsible for assigning and issueing arrival clearance to aircraft inbound to Perth via SOLUS.
 !!! note
-    Controllers should be aware there may be limited ADS-B coverage around Albany (YABA). Expect identification to be lost as aircraft descend through A100.
\ No newline at end of file
+    Controllers should be aware there may be limited ADS-B coverage around Albany (YABA). Expect identification to be lost as aircraft descend through A100.
+
+
+## Coordination
+## ENR / PH TCU
+### Standard Assignable Levels
+The Standard assignable level from ENR to PH TCU is:  
+`A090` for all arrivals.
+
+All other aircraft must be voice coordinated to PH TCU prior to **20nm** from the boundary.
+## PIY (ALL) / ENR
+
+As per the enroute coordination requirements (link), voiceless coordination exists between enroute sectors, with no changes within 20nm to the boundary.
+
+That being said, it is *advised* that BIK give **Heads-up Coordination** to the relevant sector, prior to **20nm** from the boundary.
+
```

Comment:
needs LEA/PIY / PEA APP coord section, maybe need info from raaf guys on actually filling it out

Reply:
I'll leave that for the RAAFies to advise on. Not too familiar how it works at Perth as the PEA TCU is contained in the ASA PH TCU building